test/RegexTests.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+import sys, os.path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '../src/asm2cfg'))
+
+import asm2cfg
+
+class FunctionHeaderTest(unittest.TestCase):
+    def test_unstripped(self):
+        strip, fun = asm2cfg.get_stripped_and_function_name('Dump of assembler code for function test_function:')
+        self.assertFalse(strip)
+        self.assertEqual(fun, 'test_function')
+
+    def test_stripped(self):
+        strip, fun = asm2cfg.get_stripped_and_function_name('Dump of assembler code from 0x555555555faf to 0x555555557008:')
+        self.assertTrue(strip)
+        self.assertEqual(fun, '0x555555555faf-0x555555557008')
+
```

Comment:
flake8 (PEP8) have not been configured for this repository yet but it requires two empty lines between the classes. Also some of the lines go over 120 characters which is set in `setup.cfg` file. On top of this PR flake8 tool could be configured for style checks using github actions.

Reply:
Fixed but there's still a warning about `sys.insert` (Flake wants it after imports).

+1 for adding this to CI.